client/src/App.tsx

```diff
@@ -1,7 +1,9 @@
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-function App() {
-  return <div>Hello world</div>;
-}
+import { RouterProvider } from 'react-router-dom';
+
+import { router } from './router.jsx';
+
+const App = (): JSX.Element => <RouterProvider router={router} />;
```

Comment:
This is our main page - component also. I think we should move router in main file

```suggestion
const App = (): React.FC => <something mock>;
```